staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,55 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, this directory will be skipped.
```

Comment:
actually it goes up to the parent dir until it finds one, @eloyekunle can comment

Reply:
Yes, it goes up to the parent directory.